tests/integration/targets/nxos_smoke/tasks/main.yaml

```diff
@@ -0,0 +1,14 @@
+---
+# Some of the tests in this suite change the hostname to switch.
+# This block/always ensures the hostname gets changed back to
+# the correct name.
+- block:
+  - { include: cli.yaml, tags: ['cli'] }
+  - { include: nxapi.yaml, tags: ['nxapi'] }
```

Comment:
```suggestion
  - { include: cli.yaml, tags: ['network_cli'] }
  - { include: nxapi.yaml, tags: ['httpapi'] }
```

Reply:
We have been using `cli` and `nxapi` tags in all the tests and in fact, CI uses `nxapi` tag to skip httapi/nxapi tests.

Reply:
Alright, I had assumed that change was going to be applied to both, but I suppose this isn't the PR to enforce that. Carry on.